pkg/operation/botanist/component/vpnshoot/vpnshoot.go

```diff
@@ -116,6 +120,12 @@ type Values struct {
 	ReversedVPN ReversedVPNValues
 	// Network contains the configuration values for the network.
 	Network NetworkValues
+	// VPNHighAvailabilityEnabled marks whether HA is enabled for VPN.
+	VPNHighAvailabilityEnabled bool
+	// VPNHighAvailabilitySeedServers is the number of VPN seed servers used for HA
+	VPNHighAvailabilitySeedServers int
+	// VPNHighAvailabilityShootClients is the number of VPN shoot clients used for HA
+	VPNHighAvailabilityShootClients int
```

Comment:
WDYT? The package is already about VPN-Shoot, so we can drop this?

```suggestion
	// HighAvailabilityEnabled marks whether HA is enabled for VPN.
	HighAvailabilityEnabled bool
	// HighAvailabilityNumberOfSeedServers is the number of VPN seed servers used for HA
	HighAvailabilityNumberOfSeedServers int
	// HighAvailabilityNumberOfShootClients is the number of VPN shoot clients used for HA
	HighAvailabilityNumberOfShootClients int
```

Reply:
renamed as suggested